src/app/core/pages/admin-page/admin-page.component.html

```diff
@@ -0,0 +1,208 @@
+<div class="admin">
+    <h2 class="admin__title">Create new card</h2>
+
+    <form class="admin__form" [formGroup]="form" (ngSubmit)="onSubmit()">
+        <div
+            class="admin__control"
+            [ngClass]="{
+                invalid:
+                    form.get('title')?.touched && form.get('title')?.invalid
```

Comment:
here is better to create a getter

Reply:
please, fix it in the whole file

Reply:
I made a fix for the entire file. Added getters.

```
    get title() {
        return this.form.get('title');
    }

    get description() {
        return this.form.get('description');
    }

    get imageLink() {
        return this.form.get('imageLink');
    }

    get videoLink() {
        return this.form.get('videoLink');
    }

    get creationDate() {
        return this.form.get('creationDate');
    }

   ```
     [ngClass]="{
                invalid: title?.touched && title?.invalid
     }"